libmscore/layout.cpp

```diff
@@ -2455,6 +2455,10 @@ void Score::getNextMeasure(LayoutContext& lc)
             }
 
       for (Segment& segment : measure->segments()) {
+            Segment* ns = segment.nextEnabled();
```

Comment:
Let's call it `nextSegment`. The code becomes clearer.

Reply:
nextEnabled() is not a new method. And it doesn't return nextSegment.  It return next enabled segment.